nodes/src/nodes/tool_python/IGlobal.py

```
@@ -24,46 +24,35 @@
 Python tool node - global (shared) state.
 
-Creates a ``PythonDriver`` that exposes a single ``execute`` tool for agent
-invocation.
+Reads config and stores sandbox settings (allowed modules, timeout)
+for IInstance tool methods.
 """
 
 from __future__ import annotations
 
-from typing import Set
+from typing import Optional, Set
```

Ruff (0.15.7)

[warning] `typing.Set` is deprecated, use `set` instead

(UP035)